randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -442,20 +452,28 @@ Extra - map_name: rm_a5c12
   * Layers: default
   > Door to Distribution Facility Tower West
       All of the following:
```

Comment:
This is missing the complications you have to do, when breaking the bomb blocks with only bombs. Since you cant reach them directly from the middle, you need to do mid air morph shenanigans or something similar.

Reply:
I figured since you can just spam bombs to reach the wall it was unnecessary?

Reply:
i dont think you can reach it with just spamming bombs? seems way too high up for it with its 7 tiles

Reply:
Just confirmed in game it is possible.

Reply:
Feels too much IMO, i'd prefer it if the mid air stuff was there, especially since this is potentially with a zeta in the room